CHANGELOG.md

```diff
@@ -77,6 +77,12 @@
   and `metalk8s-keepalived` image to `alpine:3.18.5`
   (PR[#4208](https://github.com/scality/metalk8s/pull/4208))
 
+- Bump ingress-nginx chart version to
+  [4.8.4](https://github.com/kubernetes/ingress-nginx/releases/tag/helm-chart-4.8.4)
+  The controller image has been bumped accordingly to
+  [v1.9.4](https://github.com/kubernetes/ingress-nginx/releases/tag/controller-v1.9.4)
+  (PR[#4211](https://github.com/scality/metalk8s/pull/4211))
```

Comment:
It was already bumped in this release so just update the previous changelog entry (just 2 above this one) to match the new version and PR